src/components/NavSection.vue

```diff
@@ -32,23 +35,31 @@
 
 <script>
 import { SET_QUERY } from '@/store/mutation-types';
+import HomeLicenseFilter from './HomeLicenseFilter';
 
 export default {
+  name: 'nav-section',
```

Comment:
This name does not match the component's name

Reply:
We have NavSection as Component's name, earlier, it used to be the same name only, i just re-ordered the sequence.